plugin/rewrite/edns0.go

```diff
@@ -274,14 +275,22 @@ func (rule *edns0VariableRule) ruleData(w dns.ResponseWriter, r *dns.Msg) ([]byt
 		return rule.portToWire(port)
```

Comment:
I think at the end, all these variables included into ruleData .. should be supplied by REWRITE (or another ?) as a metadataer.

But do not change right now ... it is better to go by step.